subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/transform/TestTransformationWorkspaceProvider.java

```diff
@@ -42,7 +41,7 @@ public ExecutionHistoryStore getExecutionHistoryStore() {
 
     @Override
     public Cache<UnitOfWork.Identity, Try<ImmutableList<File>>> getIdentityCache() {
-        return emptyCache;
+        return new ManualEvictionInMemoryCache<>();
```

Comment:
Does this need to be created on every call to the getter?

Reply:
It should be, since if it not created on every call, then it may not be empty, right?